tests/data/cli/compare/virt-install-kvm-loongarch64-basic.xml

```diff
@@ -0,0 +1,93 @@
+<domain type="kvm">
+  <name>fedora19</name>
+  <uuid>00000000-1111-2222-3333-444444444444</uuid>
+  <metadata>
+    <libosinfo:libosinfo xmlns:libosinfo="http://libosinfo.org/xmlns/libvirt/domain/1.0">
+      <libosinfo:os id="http://fedoraproject.org/fedora/19"/>
+    </libosinfo:libosinfo>
+  </metadata>
+  <memory>65536</memory>
+  <currentMemory>65536</currentMemory>
+  <vcpu>2</vcpu>
+  <os>
```

Comment:
The fact that `<os firmware='efi'>` is not used makes this looks quite unrealistic. Might be connected to the lack of EFI information in the domcapabilities?

Reply:
Yes,After I add EFI information to domcapabilities, re-execute the command “pytest --regenerate-output”,the issue will be resolved.